src/genome_builds.php

```diff
@@ -85,9 +97,11 @@
             // Raise error if ID is empty or not addable.
             lovd_errorAdd('id', 'Please select a genome build.');
         }
-
-        // TODO: Ask Ivo how to use var $aAddableGBs up OR how to use $_POST below,
-        // TODO: to be able to throw an error when a user tries to add an unaddable GB
+        elseif (in_array($_POST['id'], $aAddableGenomeBuilds)) {
```

Comment:
This check isn't correct indeed, because the ids (e.g., "hg38") are the *keys* of `$aAddableGenomeBuilds`. However, `in_array()` doesn't check the keys, but the values. I updated this and used `isset()` here to check for the existence of the key.